test/docker/ubuntu2004

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04
```

Comment:
May I suggest using
```suggestion
FROM ubuntu:latest
```
It seems to default to the latest LTS. In addition, we could add a Dockerfile for `ubuntu:devel`, which is the current Ubuntu version.

Reply:
I do not think this is a good idea. It would mean we would introduce a dynamic variable in the travis configuration that would change with every new release. I think it's better to keep it clearly distinguishable.